src/snippets/muiPopper.ts

```diff
@@ -0,0 +1,16 @@
+import snip from '../snip'
+
+export const description = 'Material-UI <Popper>'
+export const body = snip`
+const [anchorEl, setAnchorEl] = useState(null);
+const handleClick = event => setAnchorEl(anchorEl ? null : event.currentTarget);
+const open = Boolean(anchorEl);
+
+<Popper open={open} anchorEl={anchorEl} transition>
+  {({ TransitionProps }) => (
+    <Fade {...TransitionProps} timeout={350}>
```

Comment:
I think we should name this `muiPopperFade`

Reply:
Actually, since there are several different transition components, I think we should name this `muiPopperTransition` and have the snippet prompt us to select which transition component we want

Reply:
Also I've tried to avoid putting state management like this into snippets for everyone's use, since I worry n00b developers would not understand that they need to put a `return` before the popper or cut/paste it into the element they're returning.  So I might decide you have to stick with your own snippet for this, but let me think about if there are any other possible ways to solve this...

We would at least want `anchorEl`, `setAnchorEl` etc to be tab stops that people can change the name of, to avoid conflicts if they're using multiple poppers in a component